### PR TITLE
Adding Code Episode 67 - Spring for GraphQL with Dan Vega

### PR DESCRIPTION
**Pull request by danvega**

### CODE REVIEW

**Comment by danvega** [comment]
@BrianMMcClain I adding GraphQL to the custom dictionary and it's still failing spell check. What am I missing?

File: content/tv/code/0064/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+aliases:
+- /tv/code/67
+episode: '67'
+image: /tv/code/images/dan-vega.jpg
+explicit: 'no'
+hosts:
+- Dan Vega
+lastmod: '2022-04-04'
+minutes: 120
+title: "Spring for GrqphQL"
```
Comment (BrianMMcClain) [inline content/tv/code/0064/index.md:14]:
```suggestion
title: "Spring for GraphQL"
```

Comment (BrianMMcClain) [reply content/tv/code/0064/index.md:14]: @danvega Looks like this is why it was failing, there was a typo in the title. Once this change is accepted, it should pass and I can merge :)

**Review by BrianMMcClain — CHANGES_REQUESTED** [review]

**Comment by danvega** [comment]
Thanks @BrianMMcClain

**Review by BrianMMcClain — APPROVED** [review]

**Pull request merged**